react/week1/my-app/src/components/toDoList.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import Table from 'react-bootstrap/Table';
+
+class ToDoListTable extends Component {
+    render() { 
+        const rows = [];
+        this.props.toDoList.forEach(toDo => {
+            rows.push(
+                <ToDoItemRow 
+                toDo = {toDo} />
+            );
+        });
```

Comment:
Here it would be easier for you if you use the `map` function instead. It would look something like this:

```suggestion
    const rows = this.props.toDoList.map((todo) => (
      <ToDoItemRow toDo={todo} />
    ));
```